app/helpers/course/controller_helper.rb

```diff
@@ -0,0 +1,30 @@
+module Course::ControllerHelper
+  # Formats the given +CourseUser+ as a user-visible string.
+  #
+  # @param [CourseUser] user The User to display.
+  # @return [String] The user-visible string to represent the User, suitable for rendering as
+  #   output.
+  def display_course_user(user)
+    user.name
```

Comment:
Yah, but things like creator, we are actually displaying the user name in course